### PR TITLE
Add Dockerfile,init and readme file for Apim v.2.6.0

### PR DESCRIPTION
**Pull request by chamindi-a**

### CODE REVIEW

**Comment by chirangaalwis** [comment]
This fixes https://github.com/wso2/docker-apim/issues/165, fixes https://github.com/wso2/docker-apim/issues/166 and https://github.com/wso2/docker-apim/issues/167.

File: dockerfiles/alpine/apim-analytics/init.sh [hunk 1]
```diff
@@ -20,39 +20,6 @@ set -e
 config_volume=${WORKING_DIRECTORY}/wso2-config-volume
 artifact_volume=${WORKING_DIRECTORY}/wso2-artifact-volume
 
-# a grace period for mounts to be setup
-echo "Waiting for all volumes to be mounted..."
```
Comment (ThilinaManamgoda) [inline dockerfiles/alpine/apim-analytics/init.sh]: Have we figured out a way to solve the mounting problem?

Comment (chirangaalwis) [reply dockerfiles/alpine/apim-analytics/init.sh]: @chamindi-a why is the mount checking implementation removed from the Alpine based Dockerfile for APIM Analytics?

Comment (chirangaalwis) [reply dockerfiles/alpine/apim-analytics/init.sh]: @chamindi-a any particular reason this was removed?

Mount checking implementation is mandatory in Alpine and CentOS based Docker images.

**Pull request merged**